ch_backup/logic/upload_part_observer.py

```diff
@@ -0,0 +1,50 @@
+"""Uploading part observer."""
+import time
+from typing import List, Optional
+
+from humanfriendly import parse_timespan
+
+from ch_backup.backup.metadata.part_metadata import PartMetadata
+from ch_backup.backup_context import BackupContext
+
+DEFAULT_INTERVAL = parse_timespan("30 min")
```

Comment:
Looks redundant as there is a default in `config.py`.

Reply:
You are right. It was done due to one old unit test. But I have reworked it.